### ISSUE TITLE
Ideas for armor types

### ISSUE THREAD

**Issue by cdqwertz** [cdqwertz/rpgtest#16]
Post your ideas for **armor types** (diamond armor, copper armor...) here

**Comment by ObaniGemini** [comment]
I think we shall be abled to make armors with mobs drops (leather armor, minotaur skin armor,...)

Also, maybe keep the usual mineral armors ? (diamon, iron, copper, bronze, mese?, mithril?, ...)

**Comment by farfadet46** [comment]
maybe you can make armor like dirt armor to get invisible to certain mob. or you can make potion to do that ? i don't know.

**Comment by Dragonop** [comment]
You could develop the armor mod further, instead of just 3 pieces of armor per set, what about making individual parts and slots for everything. Diablo III does something similar: 
![Diablo III armor](http://www.diablowiki.net/images/4/46/Paper-doll-annotated-2010.jpg)
You could make it even more complex, making stuff like collecting unique armor sets something more challenging, or combining different armor types to get a desired effect. This would also make crafting armor not so cheap, making the game harder.

Now, armor types: 
Leather armors: Armors made out of the fur and skin from monsters with wood as reinforcement, you could start with armor made out of wood and leather from pigs and cows. This could give a good armor to start, without the risks of getting killed because you decided to fight a mob without armor.

Iron armors: Scales, chain, plate, etc.

That's all I got, thanks for reading.

**Comment by ObaniGemini** [comment]
:-1: 
It would just be a loss of time for something that would become a mess and a very complicated feature for players.
Example : 
You have 8 armors types = 24 items or 40 items (depends on the mod). With this feature : 80 items ? 104 items ?

No I think this wouldn't be useful, as armors are not a key-piece in gameplay.
You can't win if you have bad armors in Diablo, whereas you can in Minetest.

**Comment by cdqwertz** [comment]
Whats about **5** pieces of armor?

- leggings
- chestplate
- boots
- helm
- gloves

**Comment by farfadet46** [comment]
need to explain to the player what item is important and why like the glove, you need it or it's just a visual feature ?

personaly i think one item = one armour is enouth simply create only one place in the formspec

like an diamond armour is crafted by many other craft 
(the crafting is more complicated, but you can't use before you can assemble the armor)

it's just to say i don't like the system where you can use different type of armour like wood helmet + iron boots + dirt chest plate + diamond leggins

XD

**Comment by cdqwertz** [comment]
When you use more pieces of armor, it is more realistic...

**Comment by ObaniGemini** [comment]
Your game is meant to be realistic ? Minetest is meant to be realistic ? 0.o

**Comment by cdqwertz** [comment]
its not meant to be realistic, but I want to make it a bit more realistic than mt...

**Comment by farfadet46** [comment]
voted o/ 

PS: just realized there is a forum topic :'(

**Comment by Dragonop** [comment]
Yeah, I guess 5 armor pieces is good. I mean, how is it useful to have a chesplate, if your arms are unprotected?
How is a loss of time to add content to the game? After all you want your players to play for more than one day, an RPG that you can complete in a single playthrough is not deep enough, and could probably use more development.

@ObaniGemini: You can win with bad armors in Diblo, it depends on your _character type and gameplay style _(ranged, melee, spellcaster, summoner)

**Comment by cdqwertz** [comment]
what do you think about something like a jetpack?

**Comment by farfadet46** [comment]
nope i think jetpack is cheating in rpg 
In which circumstances you need it ?

**Comment by cdqwertz** [comment]
@farfadet46 It was just an idea :)

**Comment by farfadet46** [comment]
why not but need to think about it ;) like if there is some sky-island and the only way to go to it is the jetpack ;)

**Comment by cdqwertz** [comment]
@farfadet46 I think it would be great to have something like another dimension... with floating island and many glowing things...

**Comment by farfadet46** [comment]
ok for me to see another dimensions in your sub-game ;) like unicorn island XD

**Comment by cdqwertz** [comment]
I will think about dimensions again, when I finished the 1.0 version :)

**Comment by farfadet46** [comment]
the unicorn is near alive ;)
![unicorn](https://cloud.githubusercontent.com/assets/11613522/13795562/dd91fec2-eb01-11e5-9f45-5d3cdc2f4e00.PNG)
